amm/repl/src/test/scala/ammonite/DualTestRepl.scala

```diff
@@ -0,0 +1,32 @@
+package ammonite
+
+import ammonite.interp.Preprocessor
+import ammonite.util.{Evaluated, Res}
+
+/**
+  * Wraps several [[TestRepl]], and runs its tests against all of them.
+  */
+class DualTestRepl { dual =>
+
+  def predef: (String, Option[ammonite.ops.Path]) = ("", None)
+
+  val repls = Seq(
+    new TestRepl {
+      override def predef = dual.predef
+    },
+    new TestRepl {
+      override def predef = dual.predef
+      override def codeWrapper = Preprocessor.CodeClassWrapper
+    }
+  )
+
+  def interps = repls.map(_.interp)
+
+  def session(sess: String): Unit =
```

Comment:
I'm a bit suspicious of whether or not these are actually running the tests as desired on both wrapping levels, because the CI times haven't doubled as I would expect. But if you've tested it manually and it's doing the right thing that's good enough for me